docs/contributing.rst

```diff
@@ -99,6 +99,13 @@ specific failures when they happen. Also note that the full test suite
 will attempt to modify your system's Apache config if your user has sudo
 permissions, so it should not be run on a production Apache server.
 
+Running the tests on Ubuntu requires some extra packages to be installed.
+You can install them like this:
+
+.. code-block:: shell
+
+   sudo apt install libaugeas-dev python2.7-dev python3-dev
```

Comment:
Other than `python3-dev`, these dependencies are installed by following the instructions in the [Getting Started](https://certbot.eff.org/docs/contributing.html#running-a-local-copy-of-the-client) section of our developer guide. I created #4735 to track that issue.

If you wanted, you could add something to the top of this section saying something like:
```
If you're not running the client using our :ref:`Docker setup <docker>`, make sure you follow the instructions at the beginning of the :ref:`Getting Started <getting_started>` section to ensure Certbot and it's dependencies are properly installed.
```
If you did something like this, I think it should be in a separate PR though.